TestApp/src/main/AndroidManifest.xml

```diff
@@ -25,7 +25,8 @@
         <meta-data android:name="com.google.android.gms.car.application"
             android:resource="@xml/automotive_app_desc"/>
 
-        <activity android:name=".presentation.MainActivity">
+        <activity android:name=".presentation.MainActivity"
+            android:exported="true">
```

Comment:
Why do we need to mark these activities as exported ? We aren't invoking these activities from outside of the test app, are we ?

Reply:
Launch activities are technically always invoked from outside the app :)

`AudioPlayerActivity`  and `MediaButtonReceiver` get directly invoked from the notifications